### PR TITLE
Branch 1.4 fixes yupei

### PR DESCRIPTION
**Pull request by syoopie**

### CODE REVIEW

**Review by TLChicken — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,9 +3,13 @@ layout: page
 title: User Guide
 ---
 
-**NUSTracker** is a desktop application for NUS event directors and student leaders of student organizations to keep track and manage events that their members are a part of, and is optimized
-for use via the Command Line Interface (CLI). Nonetheless, it is supported by a detailed and elegant
-Graphical User Interface (GUI). This page serves as a guide inorder to get you stared with setting-up and using NUSTracker.
+**nustracker** is a desktop application for NUS event directors and student leaders of student organizations in the School of Computing to manage students and events.
+
+As an event director/student leader, you will be able to add students and events into **nustracker** and use it to keep track of which students are attending which event, and even blacklist students from certain events. These are just some of the features, and there will be even more coming in the future. **nustracker** is also customized to your needs, and is simple, intuitive and easy to pick up.
```
[the remark this answers is not in the document]
Comment (syoopie) [reply docs/UserGuide.md:8]: i think its ok to use customized

File: src/main/java/nustracker/ui/HelpWindow.java [hunk 2]
```diff
@@ -36,7 +36,8 @@
 
     public static final String EDIT_COMMAND = "edit";
     public static final String EDIT_COMMAND_EXAMPLE = " " + EDIT_COMMAND
-            + " [id/ID_OF_STUDENT_TO_EDIT] [n/NAME] [m/MAJOR] [id/NEW_STUDENT_ID] [y/YEAR] [p/PHONE] [e/EMAIL]";
+            + " [id/ID_OF_STUDENT_TO_EDIT] [n/NEW_NAME] [m/NEW_MAJOR]"
```
Comment (TLChicken) [inline src/main/java/nustracker/ui/HelpWindow.java:39]:
```suggestion
            + " id/ID_OF_STUDENT_TO_EDIT [n/NEW_NAME] [m/NEW_MAJOR]"
```
The first id for edit command is compulsory

**Review by TLChicken — APPROVED** [review]
LGTM

**Pull request merged**